mappings/net/minecraft/block/FruityLogBlock.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/class_2149 net/minecraft/block/FruityLogBlock
```

Comment:
Why did you decide on "fruity" for the name? (I copied this comment over lol)

Reply:
It matches the leaves name haha

Reply:
see comments on other discussion